src/pyodide/internal/python.js

```diff
@@ -397,6 +420,9 @@ function memorySnapshotDoImports(Module) {
 function makeLinearMemorySnapshot(Module) {
   memorySnapshotDoImports(Module);
   const dsoJSON = recordDsoHandles(Module);
+  if (IS_CREATING_BASELINE_SNAPSHOT) {
+    // checkLoadedSoFiles(dsoJSON);
```

Comment:
Should this be commented out?

Reply:
It doesn't work with the intentionally messed up creation logic in `preloadDynamicLibs` so we can leave a todo to enable a check like that.